mandatory/3-function-output.js

```diff
@@ -3,15 +3,25 @@ function getRandomNumber() {
   return Math.random() * 10;
 }
 
+//The Math.random() function returns a floating-point, pseudo-random number 
+//that's greater than or equal to 0 and less than 1, with approximately uniform 
+//distribution over that range — which you can then scale to your desired range.
+
+
 // Add comments to explain what this function does. You're meant to use Google!
 function combine2Words(word1, word2) {
   return word1.concat(word2);
 }
 
+//In JavaScript, concat() is a string method that is used to concatenate strings together.
+// The concat() method appends one or more string values to the calling
+// string and then returns the concatenated result as a new string.
+
+
 function concatenate(firstWord, secondWord, thirdWord) {
-  // Write the body of this function to concatenate three words together.
-  // Look at the test case below to understand what this function is expected to return.
-}
+  return  firstWord.concat(secondWord.concat(thirdWord))
```

Comment:
please test it

Reply:
It's done, now it looks like this:
function concatenate(firstWord, secondWord, thirdWord) {
  return  firstWord.concat(secondWord, thirdWord)
 }